projects/packages/sync/src/class-rest-endpoints.php

```diff
@@ -353,7 +353,7 @@ public static function full_sync_start( $request ) {
 				$modules['users'] = 'initial';
 			} elseif ( is_array( $request->get_param( $module_name ) ) ) {
 				$ids = $request->get_param( $module_name );
-				if ( count( $ids ) > 0 ) {
+				if ( is_countable( $ids ) && count( $ids ) > 0 ) {
```

Comment:
I note this one is redundant. We already checked `is_array()` just above on line 354.

Maybe this could use `!== array()` instead to make the tool happy.